### PR TITLE
[RFR] Publish automatically Git tag when publishing on npm

### LINKED ISSUE #162: Publish Git tag while publishing to npm

### PR DESCRIPTION
**Pull request by jpetitcolas**
Fixes #162.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,11 @@ build: install ## Webpack build the project
 	mkdir -p dist
 	./node_modules/.bin/rollup -c
 
-publish: build ## Publish current version of EventDrops
+publish: build ## Publish current version of EventDrops (need to set a VERSION=[major|minor|patch])
+ifndef VERSION
```
Comment (jpetitcolas) [inline Makefile]: We can't add some indentation here.

**Comment by djhi** [comment]
Tests are failing though

**Comment by jpetitcolas** [comment]
Tests should be fixed now.

File: Makefile [hunk 2]
```diff
@@ -11,7 +11,14 @@ build: install ## Webpack build the project
...
 
-publish: build ## Publish current version of EventDrops
+publish: build ## Publish current version of EventDrops (need to set a VERSION=[major|minor|patch])
+	@if [ -z ${VERSION} ]; then \
+		echo "Error: please set VERSION envvar to one of the following value: major, minor, or patch.\n"; \
+		exit 1; \
+	fi
+
+	npm version ${VERSION}
+	git push --tags
```
Comment (Luwangel) [inline Makefile]: You could use scripts to work with version in order to automatically push the tags for example

https://docs.npmjs.com/cli/version

``` json
"scripts": {
  "preversion": "npm test",
  "version": "npm run build && git add -A dist",
  "postversion": "git push && git push --tags && rm -rf build/temp"
}
```

Comment (jpetitcolas) [reply Makefile]: We put all commands in the `Makefile`, hence these hooks are already implemented in it. Yet, :+1: for launching tests before publishing. PR updated. :)

**Pull request merged**